prinz_sample/src/main/scala/pl/touk/nussknacker/prinz/sample/SampleConfigCreator.scala

```diff
@@ -19,7 +26,19 @@ class SampleConfigCreator extends EmptyProcessConfigCreator {
     "empty" -> allCategories(SinkFactory.noParam(EmptySink))
   )
 
-  override def services(processObjectDependencies: ProcessObjectDependencies): Map[String, WithCategories[Service]] = Map(
-    "prinzEnricher" -> allCategories(new PrinzEnricher)
-  )
+  override def services(processObjectDependencies: ProcessObjectDependencies): Map[String, WithCategories[Service]] = {
+    val repo = MLFRepository(new URL(MLFConfig.mlflowProxyUrl))
+    val response = repo.listModels
+
+    if(response.isRight) {
+      val modelsList = response.right.get
+      modelsList.foldLeft(Map.empty[String, WithCategories[Service]])(
+        (s, m) => s + (m.getName.toString -> allCategories(new PrinzEnricher(m)))
```

Comment:
let's use more expresive names, can be short but should have some more information